frontend/src/components/Header/Header.css

```diff
@@ -23,7 +23,7 @@
 
 .nav-link {
 	padding: 20px;
-	color: white;
+	color: white !important;
```

Comment:
Why are we using `!important` here? I believe `.nav-link` is a Bootstrap class, which means there's a good chance we're going to use it elsewhere. Please take the styling off this and put a custom class on these elements.